spread.yaml

```diff
@@ -9,7 +9,6 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
```

Comment:
I also don't undestand this? shouldn't we now need to run all tests both with reexec and without? to test the current snapd from trunk